tests/unit/amo/components/TestVPNPromoBanner.js

```diff
@@ -106,26 +123,86 @@ describe(__filename, () => {
     expect(root.find('.VPNPromoBanner-cta')).toHaveProp('href', href);
   });
 
+  const _clickCta = (root) => {
```

Comment:
nit: I don't think we particularly need underscore prefixes. (We don't have `_render()`)